tests/unit/postprocessing/test_PostProcessAdmin.py

```diff
@@ -20,60 +15,5 @@ def test_bad_constructor():
         _ = PostProcessAdmin()
 
 
-def test_good_constructor(mocker):
```

Comment:
let's see if we can keep this test, but just remove parts that don't make sense anymore, like where `reduce()` is called